devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -387,11 +387,24 @@ private dimmerEvents(physicalgraph.zwave.Command cmd, ep = null) {
 	def value = (cmdValue ? "on" : "off")
 	def result = [createEvent(name: "switch", value: value)]
 	if (cmdValue && cmdValue <= 100) {
+		if(isDINDimmer()){
```

Comment:
Since we expose "minimumDimmingValue" parameter, we should also consider adjusting that value for all Qubino devices that are supported by this DTH.

Reply:
Agree with Konrad. I expect the same problem will occur for all of these devices.

Reply:
Done